src/components/calcite-radio/calcite-radio.scss

```diff
@@ -0,0 +1,88 @@
+:host-context([theme="dark"]) {
+  @include calcite-theme-dark();
+}
+:host {
+  .radio {
+    border-radius: 100%;
+    box-shadow: inset 0 0 0 1px var(--calcite-ui-border-1);
+    cursor: pointer;
+    transition: all 0.15s ease-in-out;
+  }
+}
+:host([hovered]),
+:host(:not([checked])[focused]) {
+  .radio {
+    box-shadow: inset 0 0 0 2px var(--calcite-ui-blue-1);
+  }
+}
+:host([disabled]) {
+  .radio {
+    cursor: default;
+    opacity: 0.4;
+  }
+}
+:host([hovered][disabled]) {
+  .radio {
+    box-shadow: inset 0 0 0 1px var(--calcite-ui-border-1);
+  }
+}
+:host([scale="s"]) {
+  .radio {
+    height: 12px;
+    min-width: 12px;
+    max-width: 12px;
+  }
+}
+:host([scale="s"][checked]),
+:host([hovered][scale="s"][checked][disabled]) {
+  .radio {
+    box-shadow: inset 0 0 0 4px var(--calcite-ui-blue-1);
+  }
+}
+:host([scale="s"][focused][checked]) {
+  .radio {
+    box-shadow: inset 0 0 0 4px var(--calcite-ui-blue-1), 0 0 0 2px var(--calcite-ui-foreground-1),
+      0 0 0 4px var(--calcite-ui-blue-1);
+  }
+}
+:host([scale="m"]) {
+  .radio {
+    height: 14px;
+    min-width: 14px;
+    max-width: 14px;
+  }
+}
+:host([scale="m"][checked]),
+:host([hovered][scale="m"][checked][disabled]) {
+  .radio {
+    box-shadow: inset 0 0 0 5px var(--calcite-ui-blue-1);
+  }
+}
+:host([scale="m"][focused][checked]) {
+  .radio {
+    box-shadow: inset 0 0 0 5px var(--calcite-ui-blue-1), 0 0 0 2px var(--calcite-ui-foreground-1),
+      0 0 0 4px var(--calcite-ui-blue-1);
+  }
+}
+:host([scale="l"]) {
+  .radio {
+    height: 20px;
+    min-width: 20px;
+    max-width: 20px;
+  }
+}
+:host([scale="l"][checked]),
+:host([hovered][scale="l"][checked][disabled]) {
+  .radio {
+    box-shadow: inset 0 0 0 6px var(--calcite-ui-blue-1);
+  }
+}
+:host([scale="l"][focused][checked]) {
+  .radio {
+    box-shadow: inset 0 0 0 6px var(--calcite-ui-blue-1), 0 0 0 2px var(--calcite-ui-foreground-1),
+      0 0 0 4px var(--calcite-ui-blue-1);
+  }
+}
+:host([hidden]) {
```

Comment:
Out of scope: @driskull @paulcpederson @macandcheese should this be moved to `global.scss` or `_host.scss`?